layout/_macro/post.swig

```diff
@@ -271,11 +271,7 @@
           {% set COLUMN_NUMBER = 3 %}
           {% for photo in post.photos %}
             {% if loop.index0 % COLUMN_NUMBER === 0 %}<div class="post-gallery-row">{% endif %}
-              <a class="post-gallery-img fancybox"
-                 href="{{ url_for(photo) }}" rel="gallery_{{ post._id }}"
-                 itemscope itemtype="http://schema.org/ImageObject" itemprop="url">
-                <img src="{{ url_for(photo) }}" itemprop="contentUrl"/>
-              </a>
+              <img src="{{ url_for(photo) }}" itemprop="contentUrl"/>
```

Comment:
Maybe we must add `itemscope itemtype="http://schema.org/ImageObject"` here, before `itemprop="contentUrl"`? It's for schema.